crypto/bindings/conversion-proof.ts

```diff
@@ -187,7 +186,10 @@ function proofConversionPerField(
     proofFromRust(proof: WasmProverProof): ProverProof {
       let commitments = commitmentsFromRust(proof.commitments);
       let openingProof = openingProofFromRust(proof.proof);
-      let evals: ProofEvaluations<Uint8Array> = proof.evals;
+      let evals = proofEvaluationsFromRust(
+        // TODO typed as `any` in wasm-bindgen, this is the correct type
+        proof.evals satisfies ProofEvaluations<Uint8Array>
```

Comment:
Can't we get rid of this? Why wasm-bindgen type it as any?

Reply:
I don't know... here is the type definition on the Rust side: https://github.com/o1-labs/o1js-bindings/blob/d185e8609205aa820eb01c7317e4b48e3c0d7d53/kimchi/wasm/src/plonk_proof.rs#L474-L476

Reply:
Let's leave it like this for the moment. Opening followup issue.